src/index.json

```diff
@@ -275,6 +275,49 @@
                     "version": "0.4.37"
                 },
                 "sha256Digest": "80d04f0d477b987dd462e6c1fb3fdbe7441c7ea1e3c75fb8593847d43b4a1e3b"
+            },
+            {
+                "downloadUrl": "https://azurecliprod.blob.core.windows.net/cli-extensions/aks_preview-0.4.38-py3-none-any.whl",
+                "filename": "aks_preview-0.4.38-py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "azext.minCliCoreVersion": "2.0.49",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.6",
+                        "Programming Language :: Python :: 3.7",
+                        "Programming Language :: Python :: 3.8",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "azpycli@microsoft.com",
```

Comment:
Shall contact email need to change ? https://github.com/Azure/azure-cli-extensions/pull/1435#discussion_r396827406

Reply:
we can keep azpycli